src/js/components/AppListComponent.jsx

```diff
@@ -16,9 +16,16 @@ var AppListComponent = React.createClass({
   },
 
   getInitialState: function () {
+    var fetchState = States.STATE_LOADING;
+    var apps = AppsStore.apps;
+
+    if (apps.length > 0) {
+      fetchState = States.STATE_SUCCESS;
+    }
```

Comment:
We had a decision in the past not to use ternary operators for better readability. This here could be an edge case, but I think we should stay in line.